tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -482,36 +483,37 @@ struct mount {
+BPF_HASH(traced_pids_map, u32, u32);                    // Keep track of traced pids
+BPF_HASH(new_pids_map, u32, u32);                       // Keep track of the processes of newly executed binaries
+BPF_HASH(new_pidns_map, u32, u32);                      // Keep track of new pid namespaces
+BPF_HASH(pid_to_cont_id_map, u32, container_id_t);      // Map pid to container id
+BPF_HASH(args_map, u64, args_t);                        // Persist args info between function entry and return
+BPF_HASH(ret_map, u64, u64);                            // Persist return value to be used in tail calls
+BPF_HASH(inequality_filter, u32, u64);                  // Used to filter events by some uint field either by < or >
+BPF_HASH(uid_filter, u32, u32);                         // Used to filter events by UID, for specific UIDs either by == or !=
+BPF_HASH(pid_filter, u32, u32);                         // Used to filter events by PID
+BPF_HASH(mnt_ns_filter, u64, u32);                      // Used to filter events by mount namespace id
+BPF_HASH(pid_ns_filter, u64, u32);                      // Used to filter events by pid namespace id
+BPF_HASH(uts_ns_filter, string_filter_t, u32);          // Used to filter events by uts namespace name
+BPF_HASH(comm_filter, string_filter_t, u32);            // Used to filter events by command name
+BPF_HASH(bin_args_map, u64, bin_args_t);                // Persist args for send_bin funtion
+BPF_HASH(sys_32_to_64_map, u32, u32);                   // Map 32bit syscalls numbers to 64bit syscalls numbers
+BPF_HASH(params_types_map, u32, u64);                   // Encoded parameters types for event
+BPF_HASH(params_names_map, u32, u64);                   // Encoded parameters names for event
+BPF_HASH(sockfd_map, u32, u32);                         // Persist sockfd from syscalls to be used in the corresponding lsm hooks
+BPF_HASH(process_tree_map, u32, u32);                   // Used to filter events by the ancestry of the traced process
+BPF_LRU_HASH(sock_ctx_map, u64, net_ctx_ext_t);         // Socket address to process context
+BPF_LRU_HASH(network_map, local_net_id_t, net_ctx_t);   // Network identifier to process context
+BPF_ARRAY(file_filter, path_filter_t, 3);               // Used to filter vfs_write events
+BPF_ARRAY(string_store, path_filter_t, 1);              // Store strings from userspace
+BPF_PERCPU_ARRAY(bufs, buf_t, MAX_BUFFERS);             // Percpu global buffer variables
+BPF_PERCPU_ARRAY(bufs_off, u32, MAX_BUFFERS);           // Holds offsets to bufs respectively
+BPF_PROG_ARRAY(prog_array, MAX_TAIL_CALL);              // Used to store programs for tail calls
+BPF_PROG_ARRAY(sys_enter_tails, MAX_EVENT_ID);          // Used to store programs for tail calls
+BPF_PROG_ARRAY(sys_exit_tails, MAX_EVENT_ID);           // Used to store programs for tail calls
+BPF_STACK_TRACE(stack_addresses, MAX_STACK_ADDRESSES);  // Used to store stack traces
```

Comment:
Please rebase

Reply:
To be honest I don't understand what's going on here, the spacing is the same, they're just moved down a line becuase of the new CONFIG variable and the new BPF_HASH. git is just being verbose?

Reply:
check tabs versus spaces.. sometimes your editor has replaced tabs/spaces for keeping all // aligned.

Reply:
Nah that wasn't the issue. Just a combination of all of the lines moving down one and adding a line in the middle. No other difference in tabs/spaces or spaces between code and comment. I put the new BPF_HASH on the bottom to prove it but can revert that so all the filter BPF_HASH maps are next to each other